diff --git a/app/Http/Kernel.php b/app/Http/Kernel.php
index 0faf218..1c0b3d1 100644
--- a/app/Http/Kernel.php
+++ b/app/Http/Kernel.php
@@ -52,7 +52,8 @@ class Kernel extends HttpKernel
         'api' => [
             'header.www.authenticate', // Centralize WWW-Authenticate handling for API 401 responses
             'auth.dynamic.once', // Unified: Bearer (OAuth) OR Basic Auth — auto-detected per request
             'auth.disabled',
-            'throttle:api',
+            //'throttle:api',
+            //'throttle:600000,1',
             'permission:read-api',
             'company.identify',
             'bindings',
diff --git a/app/Traits/Plans.php b/app/Traits/Plans.php
index 02ce4d575..18361ed49 100644
--- a/app/Traits/Plans.php
+++ b/app/Traits/Plans.php
@@ -89,6 +89,26 @@ trait Plans
     {
         $key = 'plans.limits';
 
+            $data = json_decode(<<<JSON
+                {
+                    "user": {
+                        "action_status": true,
+                        "view_status": true,
+                        "message": ""
+                    },
+                    "company": {
+                        "action_status": true,
+                        "view_status": true,
+                        "message": ""
+                    },
+                    "invoice": {
+                        "action_status": true,
+                        "view_status": true,
+                        "message": ""
+                    }
+                }
+                JSON);
+            return $data;
         return Cache::remember($key, Date::now()->addHour(), function () {
             $url = 'plans/limits';
 
