diff --git a/app/Http/Kernel.php b/app/Http/Kernel.php
--- a/app/Http/Kernel.php
+++ b/app/Http/Kernel.php
@@ -49,7 +49,7 @@ class Kernel extends HttpKernel
             'header.www.authenticate', // Centralize WWW-Authenticate handling for API 401 responses
             'auth.dynamic.once', // Unified: Bearer (OAuth) OR Basic Auth — auto-detected per request
             'auth.disabled',
-            'throttle:api',
+            // 'throttle:api',
             'permission:read-api',
             'company.identify',
             'bindings',
diff --git a/app/Traits/Plans.php b/app/Traits/Plans.php
--- a/app/Traits/Plans.php
+++ b/app/Traits/Plans.php
@@ -88,6 +88,25 @@ trait Plans
     {
         $key = 'plans.limits';
 
+        return (object) [
+            'user' => (object) [
+                'action_status' => true,
+                'view_status' => true,
+                'message' => '',
+            ],
+            'company' => (object) [
+                'action_status' => true,
+                'view_status' => true,
+                'message' => '',
+            ],
+            'invoice' => (object) [
+                'action_status' => true,
+                'view_status' => true,
+                'message' => '',
+            ],
+        ];
+
         return Cache::remember($key, Date::now()->addHour(), function () {
             $url = 'plans/limits';
 
